L/libexif/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "libexif"
+version = v"0.6.21"
+
+# Collection of sources required to complete build
+sources = [
+    "https://jaist.dl.sourceforge.net/project/libexif/libexif/0.6.21/libexif-0.6.21.tar.gz" =>
+    "edb7eb13664cf950a6edd132b75e99afe61c5effe2f16494e6d27bc404b287bf",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libexif-*/
+if [[ "${target}" == powerpc64le-* ]] || [[ "${target}" == *-freebsd* ]]; then
+    # Install `autopoint` and other tools needed by `autoreconf`
```

Comment:
can `autoreconf` etc. be pre-installed in the vm?

Reply:
It's already available in the environment, not sure what you're referring to.

Reply:
sorry, the `autopoint`

Reply:
I think this is the first time I needed to use this tool, I don't think it's worth adding it by default.